ui/components/layouts/dashboard.js

```diff
@@ -141,7 +141,13 @@ function Layout({ children }) {
     },
   ]
 
-  const subNavigation = [{ name: 'Account', href: '/account' }]
+  const subNavigation = [
+    {
+      name: 'Account',
+      href: '/account',
+      show: auth?.providerNames.includes('infra'),
```

Comment:
While rare `providerNames` has been empty/null in the past, we may want to guard that:

```suggestion
      show: auth?.providerNames?.includes('infra'),
```